src/bootstrap/lib.rs

```diff
@@ -210,6 +210,16 @@ pub struct Compiler {
     host: Interned<String>,
 }
 
+#[derive(PartialEq, Eq, Copy, Clone, Debug)]
+pub enum DocTestsOption {
```

Comment:
Can we rename this to `DocTests`? I think it might have uses outside just being passed on the command line...